src/bundle/Resources/public/scss/_balloon-form.scss

```diff
@@ -26,11 +26,104 @@
+                        }
+                    }
+
+                    .ck-list {
+                        padding: calculateRem(4px);
+
+                        &__item {
+                            border-radius: $ibexa-border-radius;
+
+                            &:hover {
+                                color: $ibexa-color-dark;
+                                background-color: $ibexa-color-light-300;
+
+                                .ck-button {
+                                    background-color: transparent;
+                                }
+                            }
+                        }
+                    }
+
+                    .ck-button.ck-dropdown__button {
+                        .ck-button {
+                            &__label {
+                                width: calculateRem(198px);
+                                display: inline-block;
+                            }
+                        }
+                    }
+
+                    .ck-dropdown__panel--multiple {
+                        .ck.ck-button {
+                            &__toggle {
+                                display: none;
+                            }
+
+                            .ibexa-ckeditor-input--checkbox {
+                                appearance: none;
+                                position: relative;
+                                display: inline-block;
+                                cursor: pointer;
+                                outline: none;
+                                border: calculateRem(1px) solid $ibexa-color-dark-300;
+                                width: calculateRem(16px);
+                                height: calculateRem(16px);
+                                border-radius: calculateRem(2px);
+                                margin-right: calculateRem(16px);
+
+                                &::after {
+                                    content: ' ';
+                                    position: absolute;
+                                    top: calculateRem(3px);
+                                    left: calculateRem(3px);
+                                    display: block;
+                                    width: calculateRem(8px);
+                                    height: calculateRem(5px);
+                                    border-left: calculateRem(2px) solid transparent;
+                                    border-bottom: calculateRem(2px) solid transparent;
+                                    transform: rotate(-45deg);
+                                }
+
+                                &:hover {
+                                    border-color: $ibexa-color-primary;
+                                }
+
+                                &:focus {
+                                    border-color: $ibexa-color-primary;
+                                    box-shadow: 0 0 0 calculateRem(4px) rgba($ibexa-color-primary, 0.25);
```

Comment:
this will be the third use of such box-shadow i the project. What do you think about creating new shadow variable?